configure.ac

```diff
@@ -3828,11 +3828,11 @@ AS_CASE(["$target_os"],
 	    AC_LANG_BOOL_COMPILE_TRY([
 @%:@include <AvailabilityMacros.h>
 enum {
-    least = MAC_OS_X_VERSION_10_7, /* just fail if undefined */
+    least = 1070, /* just fail if undefined */
     required = MAC_OS_X_VERSION_MIN_REQUIRED,
     upper /* bigger than MIN_REQUIRED, or */
 @%:@ifdef MAC_OS_X_VERSION_10_10
-        = MAC_OS_X_VERSION_10_10
+        = 101000
```

Comment:
@nobu They are, in AvailabilityMacros.h. MacPorts patches these cases: https://github.com/macports/macports-ports/blob/master/lang/ruby31/files/patch-configure.diff

However, I am not insisting on this, if there are reasons not to use a standard convention.

Reply:
Are the macros unavailable on some platforms? i.e. `MAC_OS_X_VERSION_10_10` is not available on older versions of macOS right?

Reply:
@ioquatix Yes, but there is `ifdef`, so it won’t be invoked on old systems anyway, no? As a matter of fact, 10.5.8 and 10.6 build fine with magic numbers used in configure.

(But, as I said, I am not insisting here. I am pretty sure magic numbers are standard, since I’ve seen them across dozens of ports. But probably it is okay without them too.)

Reply:
I think what's not clear is whether there is an advantage to magic numbers, otherwise I think we prefer the symbolic constants for readability. "101000" doesn't mean as much as the symbolic `MAC_OS_X_VERSION_10_10` (ignoring X meaning 10 for the moment :).

Reply:
I agree that magic numbers do not help - especially since they changed from XXYY  to XXYYZZ at 10.10. 
However, presumably, the issue is that the later identifiers are not present in earlier SDKs? (so. e.g. 10.5 SDK would not know about 10.10 identifiers?)